README.md

```diff
@@ -190,7 +190,7 @@ If you're using `DistributedDataParallel`, make the following changes:
 +  # Rank and world size are inferred from the XLA device runtime
 +  dist.init_process_group("xla", init_method='xla://')
 +
-+  model.to(xm.xla_device())
++  model.to(torch_xla.device())
 +  ddp_model = DDP(model, gradient_as_bucket_view=True)
```

Comment:
Is this addition of `gradient_as_bucket_view=True` intended?

Reply:
That's a false alarm. The GH is highlighting diff because the code snippet has the + & -